susemanager-utils/susemanager-sls/salt/services/docker.sls

```diff
@@ -15,7 +15,7 @@ mgr_install_docker:
 {%- endif %}
 {%- if grains['saltversioninfo'][0] >= 2018 %}
       - python3-salt
-    {%- if salt['pkg.info_available']('python', 'python2') and salt['pkg.info_available']('python2-salt') %}
+    {%- if salt['pkg.info_available']('python', 'python2') and salt['pkg.info_available']('python2-salt') and salt['pkg.info_available']('python-Jinja2', 'python2-Jinja2') %}
```

Comment:
`salt['pkg.info_available']('python-Jinja2', 'python2-Jinja2')`

So it works with both SLE15 and SLE12:
```
SLE-Module-Python2_15-SP1_x86_64/_channel:    <binary name="python2-Jinja2" package="python-Jinja2" supportstatus="l3"/>
SLE-Module-Basesystem_15_x86_64/_channel:    <binary name="python2-Jinja2" package="python-Jinja2" supportstatus="l3"/>
SLE-Manager-Tools_12_x86_64/_channel:    <binary package="python-Jinja2"             name="python-Jinja2"                    supportstatus="l3"/>
```